app/_kong_plugins/datakit/examples/convert-json-to-xml-and-back.yaml

```
@@ -0,0 +1,45 @@
+description: |
+  Use the Datakit plugin to transform JSON request bodies into XML before sending them to an external service, then convert the XML response back into JSON.
+
+extended_description: |
+  Use the Datakit plugin to transform JSON request bodies into XML before sending them to an external service, then convert the XML response back into JSON.
+
+  This example contains the following nodes:
+  1. The node `JSON_XML` converts the incoming JSON request body into XML format.
+  2. The node `CALL` sends the XML data to an external API.
```

worth mentioning it is a SOAP service

```suggestion
  2. The node `CALL` sends the XML data to an external SOAP API.
```